pandas/core/indexes/base.py

```diff
@@ -522,6 +522,9 @@ def _simple_new(cls, values, name=None, dtype=None, **kwargs):
                     values = cls(values, name=name, dtype=dtype,
                                  **kwargs)._ndarray_values
 
+        if isinstance(values, (ABCSeries, cls)):
+            values = np.asarray(values._values)
```

Comment:
this should never copy, right? 

can you add a comment about why this is needed here

Reply:
Right. Added.